apps/emqx_exhook/src/emqx_exhook_sup.erl

```diff
@@ -32,6 +32,7 @@
     id => Mod,
     start => {Mod, start_link, Args},
     type => Type,
+    %% long timeout for emqx_exhook_mgr
```

Comment:
A safer approach is to pass down a `deinit` timeout in `Args` to child. and use the same timeout * 2 for shutdown .